tests/integration/thrift/test_tornado_client.py

```diff
@@ -22,7 +22,17 @@
 
 import pytest
 
-from tchannel.tornado import TChannel
+from tchannel import TChannel
+from tchannel import thrift
+
+
+@pytest.fixture
+def thrift_service():
+    # TODO replace global thrift_service fixture that uses thrift --gen.
```

Comment:
yup - let's do a cleanup in another PR, there's still some cruft left from apache thrift hangin around, ex: "ag data.generated"